### ISSUE TITLE
FutureWarning: `distplot` is a deprecated function

### ISSUE THREAD

**Issue by dynoweb** [ranaroussi/quantstats#64]
I get this error when running some of the plots functions

qs.reports.html(stock, "SPY") or qs.plots.histogram(stock) and potentially others.

....\Anaconda3\envs\jupyter\lib\site-packages\seaborn\distributions.py:2551: FutureWarning: `distplot` is a deprecated function and will be removed in a future version. Please adapt your code to use either `displot` (a figure-level function with similar flexibility) or `histplot` (an axes-level function for histograms).
  warnings.warn(msg, FutureWarning)

**Comment by scollay** [comment]
Same here. Python 3.8.

**Comment by kartiksubbarao** [comment]
I ran into the same issue. As a workaround, I replaced the call to `_sns.distplot()` in `_plotting/core.py` with this:
```
_sns.histplot(returns, bins=bins, color=colors[0], alpha=0.7,
              kde=kde, line_kws=dict(linewidth=2), ax=ax)
```
It's not an exact match, but reasonably close. I didn't see an easy way to change the color of the density curve to be different than the bars, so I increased the linewidth to make it more prominent.

**Comment by ranaroussi** [comment]
Fixed in 0.0.34
